torch_xla/csrc/aten_xla_type.h

```diff
@@ -501,6 +501,9 @@ class AtenXlaType {
   static std::tuple<at::Tensor, at::Tensor> log_sigmoid_forward(
       const at::Tensor& self);
 
+  static at::Tensor logsumexp(const at::Tensor& self, at::IntArrayRef dim,
+                              bool keepdim);
+
   static at::Tensor logdet(const at::Tensor& self);
```

Comment:
Can you re-order `logdet()` given that you are around here?
Should be moved up some place, to keep alphabetically ordered.